templates/zerodb/zerodb.py

```diff
@@ -31,6 +31,16 @@ def _deploy(self):
         zerodb_sal.deploy()
         self.data['ztIdentity'] = zerodb_sal.zt_identity
 
+    def _monitor_disk(self):
+        self.logger.info('Monitor zerodb disk %s' % self.name)
+        disk = self._node_sal.client.system(
+            "dd if=/dev/urandom bs=1k count=1 of={}/monitor".format(self.data["path"])
```

Comment:
hard coding a value for the file name is dangerous, what is there is actually a real file named the same ?
Probably better to generate a name, check that it doesn't exists and then write to it.
and don't forget to clean up after you

Reply:
i changed file name to ```_monitor_write_disk_test``` i think that's unique enough, I had a discussion with @muhamadazmy  we don't prefer to create a new file each time and delete it to avoid doing unwanted operations and avoid having failures in deleting or so ..